config/locales/en.yml

```diff
@@ -766,6 +766,7 @@ en:
 
   errors:
     messages:
+      host_label_too_long: is too long for one or more labels of the host (maximum is %{count} characters)
```

Comment:
This message will be completely enigmatic if shown in the creation of a product because the system name is too long.

Reply:
That's not what the error would be, but because it uses the ServiceCreator.
It should be done in a cleaner way. I will definitely clean that :smile: